OpenKh.Kh2/CoctLogical.cs

```diff
@@ -203,9 +142,9 @@ public class CoctCollision
             public short Vertex2 { get; set; }
             public short Vertex3 { get; set; }
             public short Vertex4 { get; set; }
-            public short Co5Index { get; set; }
-            public short Co6Index { get; set; }
-            public short Co7Index { get; set; }
+            public Plane Plane { get; set; }

```

Comment:
`CoctLogical` and its independent child-classes are for mainly WPF MVVM binding support purpose.
`CoctPlane` is class (object), and `Plane` is struct (ValueType).
ValueType may not be suitable for WPF binding usage, because the writing to properties (like X, Y, Z) will not be reflected.
If we discontinue _OpenKh.Tools.Kh2MapCollisionEditor_ (using WPF) in near future, this is possible decision.

Reply:
Kh2MapCollisionEditor will be discontinued at some point. Right now it might still be useful for debugging purposes, so I will restore it as you suggested 👍